kythe/go/platform/tools/entrystream/entrystream.go

```diff
@@ -85,6 +92,24 @@ func main() {
 		} else {
 			rd = stream.NewJSONReader(in)
 		}
+	} else if *readRiegeli {
+		rd = func(emit func(*spb.Entry) error) error {
+			r := riegeli.NewReader(in)
+			for {
+				rec, err := r.Next()
+				if err == io.EOF {
+					return nil
+				} else if err != nil {
+					return err
+				}
+				var e spb.Entry
+				if err := proto.Unmarshal(rec, &e); err != nil {
+					return err
+				} else if err := emit(&e); err != nil {
```

Comment:
You could get rid of the else since you are returning in the preceding if

Reply:
I don't think the style guide says anything about this case.  Most of our codebase joins the lines with an `else` like this, though.